Add website content (#5)
ros-infrastructure/infrastructure-website: content/index.md

---

> title: OSRA Infrastructure Project

Reviewer: ```suggestion
title: Open Robotics Infrastructure Project
```

This stands out to me since I prefer sentence case for titles but each of these words in the title is part of the proper noun "Open Robotics Infrastructure Project" 🪦ORIP?

---

> Welcome to the Open Robotics Infrastructure Project Site.

Reviewer: ```suggestion
Welcome to the Open Robotics Infrastructure Project.
```

I'm torn about this welcome message. It reminds me a lot of old internet and so I'm fond of it, but it doesn't actually add anything. In any case I think Site is a little redundant.

> The Infrastructure project is responsible for maintaining and improving the essential tools and systems that underpin the entire ROS and Gazebo ecosystem by providing essential infrastructure. This includes managing the build servers, continuous integration platforms, and various online resources that developers and users rely on to create and utilize robotics software. By ensuring the smooth operation of these critical components, the Infrastructure project plays a vital role in supporting the development and growth of the robotics community.

Reviewer: ```suggestion
This project, chartered as part of the [OSRA](https://osralliance.org), supports the Open Robotics developers and community by providing essential infrastructure.
This includes managing the build servers, continuous integration platforms, and various online resources that developers and users rely on to create and utilize robotics software.
```

Rephrasing suggestion for the first sentence and also a switch to "semantic line breaks" aka one sentence per line.

> # How to Contribute

Reviewer: ```suggestion
# How to contribute
```

Let's standardize on "Sentence case for titles"

Want to contribute to the OSRA Infrastructure project? 
We welcome contributions of all kinds! 
Whether you're a seasoned developer or a curious enthusiast, there are opportunities for everyone.
Get involved by reporting issues, suggesting improvements, or diving into the codebase. 

> To learn more about how to contribute, check out our contribution guidelines on the project's 
> [GitHub repositories](/repos) or join our community forums for discussions and support: [discourse.ros.org](https://discourse.ros.org/), [OSRF Community Discord](https://discord.com/invite/RtJeHT8mXQ).

Reviewer: ```suggestion
To learn more about how to contribute, check out the contribution guidelines on each project's [GitHub repositories](/repos) or join our community forums for discussions and support: [discourse.ros.org](https://discourse.ros.org/), [OSRF Community Discord](https://discord.com/invite/RtJeHT8mXQ).
```

Small changes to phrasing but I also consolidated onto one line for semantic line breaks.

If you would like to know more about the project, please check out the following resources:

>   <a class="btn btn-primary" role="button" target="_blank" href="https://osralliance.org/staging/wp-content/uploads/2024/03/OSRA-explainer.pdf">OSRA Explainer</a>
>   <a class="btn btn-primary" role="button" target="_blank" href="https://osralliance.org/staging/wp-content/uploads/2024/03/infrastructure_project_charter.pdf">Infrastructure Project Charter</a>

Reviewer: Use markdown bullets instead of buttons.

```suggestion
* [The Open Source Robotics Alliance (OSRA) Explained](https://osralliance.org/staging/wp-content/uploads/2024/03/OSRA-explainer.pdf)
* [Infrastructure project charter](https://osralliance.org/staging/wp-content/uploads/2024/03/infrastructure_project_charter.pdf)
```
